src/components/Timestamp/Timestamp.tsx

```diff
@@ -28,16 +28,20 @@ const timeStyles = css`
 	white-space: nowrap;
 `;
 
+const dateFormatter = (epoch: number) =>
+	timeAgoLib(epoch, { verbose: true, daysUntilAbsolute: 1 });
+
 export const Timestamp = ({
 	isoDateTime,
 	webUrl,
 	commentId,
 	onPermalinkClick,
 }: Props) => {
-	let [timeAgo, setTimeAgo] = useState(dateFormatter(isoDateTime));
+	const epoch = Date.parse(isoDateTime);
+	const [timeAgo, setTimeAgo] = useState(dateFormatter(epoch));
 
 	useInterval(() => {
-		setTimeAgo(dateFormatter(isoDateTime));
+		setTimeAgo(dateFormatter(epoch));
```

Comment:
```suggestion
		timeAgoLib(epoch, { verbose: true, daysUntilAbsolute: 1 });
```

Might as well inline?

Reply:
You have to inline twice and we don’t have any testing of the `setInterval` working as expected, so I opted for abstracting that away.

Reply:
I actually refactored to keep the changes inside the `dateFormatter`, which means we can keep this a patch change.